### ISSUE TITLE
Windows 10 PowerShell And Symbols (←[1m, etc.)

### ISSUE THREAD

**Issue by StartsMercury** [gorilla-devs/ferium#119]
## Description

![image](https://user-images.githubusercontent.com/89975834/172376357-639a69c6-9012-4956-b184-67167cfcfe0e.png)

## To Reproduce

 0. Be on Windows 10
 1. Open PowerShell
 2. Run ferium
 3. Run ferium with the subcommand `list`

## Software version
- OS Windows
- Ferium Version 4.1.1

## Additional information
*Maybe it is not working because PowerShell was not configured for this?*
Maybe either fixing or removing the characters for just PowerShell?

**Comment by theRookieCoder** [comment]
Yes this is a know 'issue', the thing is Windows Console Host (which is used for the command prompt and powershell apps) does not support coloured output. I understand that the output on conhost looks terrible, but honestly, everyone should just use [Windows Terminal](https://github.com/microsoft/terminal).
